docs/basic-features/static-file-serving.md

```diff
@@ -18,4 +18,4 @@ export default MyImage
 
 > Don't name the `public` directory anything else. The name can't be changed and is the only directory that **Next.js** uses to serve static assets.
 
-> If you ever add a static asset that conflicts with the name of a page in the `pages` directory, the public file will be ignored in favor of the file in `pages`.
+> Make sure you don’t have static assets in conflict with files in the `pages` directory, as this will result in an error.
```

Comment:
```suggestion
> Make sure you don’t have static assets in conflict with files in the `pages` directory, as this will result in an error. An example of a conflict would be `public/hello` and `pages/hello.js` but `public/hello.txt` and `pages/hello.js` would not be as the `.txt` extension prevents them from conflicting
```
It might be good to show an example of a conflict so they understand what to avoid or we could link the err.sh here https://github.com/zeit/next.js/blob/canary/errors/conflicting-public-file-page.md